src/sourmash/sourmash_args.py

```diff
@@ -393,14 +393,27 @@ def load_pathlist_from_file(filename):
     try:
         with open(filename, 'rt') as fp:
             file_list = [ x.rstrip('\r\n') for x in fp ]
-
-        if not os.path.exists(file_list[0]):
-            raise ValueError("first element of list-of-files does not exist")
+        if len(file_list) == 0:
+            raise ValueError("pathlist is empty")
+        badFormat = 0
+        duplicate = []
+        for i in range(len(file_list)):
```

Comment:
a few comments -

* duplicate should be a `set`, not a `list`; lists are linear time traversal, sets are random lookups based on hash value.
* `badformat` could be better used as a boolean, so `badFormat = False`;
* then you can use `if badFormat`;
* the for loop can also be `for checkfile in file_list`, no need to use range;

Reply:
Okay, I'll fix all of that!